sdk/python/tests/integration/feature_repos/repo_configuration.py

```diff
@@ -570,7 +570,7 @@ def construct_test_environment(
         if isinstance(test_repo_config.online_store, dict)
         else test_repo_config.online_store
     )
-    if online_store in ["milvus", "pgvector", "qdrant", "elasticsearch"]:
+    if online_store in ["milvus", "pgvector", "qdrant", "elasticsearch", "redis"]:
```

Comment:
do we need this if block now since default version is 3?

Reply:
I havent tested all the stores for v3 for backward compatibility purpose and hence we still have v2 serialization/deserialization for the customers who want to re-serialize their existing data.